src/app/pages/ProfileScreen/ProfileScreen.tsx

```diff
@@ -21,7 +18,6 @@ class ProfileScreen extends PureComponent<ProfileProps> {
         <ProfileTitle>
           {intl.formatMessage({ id: 'profile.title' })}
         </ProfileTitle>
-        <Text>{Config.TYPE}</Text>
```

Comment:
@lukaszblasz ?

Reply:
Yep this was added to have proper information on screen what version was loaded. Please keep it for now.